scripts/unit_test.sh

```diff
@@ -1,12 +1,16 @@
 #!/bin/bash
 
+if [[ -z "$ghprbPullId" ]]; then
+  echo "Not running from a context of a PR"
+  exit 1
+fi
+
```

Comment:
use `is_pr_or_mr_build` from `build_deploy_common`

Reply:
How do I pull in build_deploy_common here? By calling `source`?

Reply:
Also
```
shellcheck scripts/unit_test.sh                                                                                        

In scripts/unit_test.sh line 11:
IMAGE_TAG="pr-${ghprbPullId}-$(git rev-parse --short=7 HEAD)"
              ^------------^ SC2154 (warning): ghprbPullId is referenced but not assigned.

For more information:
  https://www.shellcheck.net/wiki/SC2154 -- ghprbPullId is referenced but not...

```